service/controller/chart/v1/resource/release/current.go

```diff
@@ -35,6 +35,15 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
+	if releaseContent.Status == "FAILED" && releaseContent.Name == "chart-operator" {
```

Comment:
```suggestion
	if releaseContent.Status == "FAILED" && releaseContent.Name == r.projectName {
```

We can remove the hardcoding by using the project name. It just needs adding to the resource wiring.